.github/workflows/test.yml

```diff
@@ -50,11 +50,14 @@ jobs:
     needs: changes
     runs-on: ubuntu-latest
     if: ${{ needs.changes.outputs.changes == 'true' }}
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
AFAIK numba is not yet available on 3.11 but that should not stop us.
Looking forward to all these speed ups

Reply:
There's another PR for 3.11. We could test support for ot t but would need to tweak jobs so that they run all tests but the numba ones.

I would rather do it in that PR. https://github.com/pymc-devs/pytensor/pull/198

Pushes there are welcome